kafkacat/Dockerfile.ubi8

```diff
@@ -24,18 +24,18 @@ FROM ${DOCKER_UPSTREAM_REGISTRY}confluentinc/cp-base-new:${DOCKER_UPSTREAM_TAG}
 WORKDIR /build
 
 ENV VERSION=1.5.0-1
-ENV BUILD_PACKAGES="which git make cmake gcc-c++ zlib-devel curl curl-devel openssl-devel openldap-devel cyrus-sasl-devel pkgconfig lz4-devel libzstd-devel" 
+ENV BUILD_PACKAGES="which git make cmake gcc-c++ zlib-devel curl curl-devel openssl-devel openldap-devel cyrus-sasl-devel pkgconfig lz4-devel wget tar"
 
 USER root
 
 RUN echo "Building kafkacat ....." \
     && dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm \
-    && yum -q -y update \
-    && yum -q install -y $BUILD_PACKAGES \
-    && yum clean all \
-    && git clone https://github.com/edenhill/kafkacat \
+    && dnf -y update \
+    && dnf install -y $BUILD_PACKAGES \
+    && dnf clean all \
+    && git clone https://github.com/andrewegel/kafkacat \
```

Comment:
Revert to edenhill/kafkacat ..

Reply:
You never fixed your fork - I changed this back to yours, and its still failing:

```
10:18:30  [INFO] /usr/bin/ld: cannot find -lzstd
10:18:30  [INFO] 
10:18:30  [INFO] collect2: error: ld returned 1 exit status
10:18:30  [INFO] 
10:18:30  [INFO] make: *** [mklove/Makefile.base:151: kafkacat] Error 1
10:18:30  [INFO] 
[2020-05-28T17:18:31.713Z] [ERROR] The command '/bin/sh -c echo "Building kafkacat ....."     && dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm     && dnf -y update     && dnf install -y $BUILD_PACKAGES     && dnf clean all     && git clone https://github.com/edenhill/kafkacat     && cd kafkacat     && git checkout origin/master     && ./bootstrap.sh     && make     && ldd kafkacat' returned a non-zero code: 2
```

Either you:
1) Fix your fork in your own way, or
2) Approve the PR I put in: https://github.com/edenhill/kafkacat/pull/242